docs/source/portfolio_optimisation/risk_estimators.rst

```diff
@@ -459,6 +517,10 @@ Example Code
                                                              detone=True, market_component=1,
                                                              kde_bwidth=kde_bwidth)
 
+    # Finding the Hierarchical Clustering Filtered Correlation matrix
+    hierarchical_filtered = risk_estimators.filter_corr_hierarchical(cov_matrix, method='complete',
+                                                             draw_plot=False')
```

Comment:
Please add a few spaces here in line 522 so that it matches the style requirements.